pkg/controller/builder/builder.go

```diff
@@ -44,7 +44,7 @@ func createBuilderObjects(req router.Request, resp router.Response) (string, str
 		forNamespace = builder.Namespace
 	}
 
-	objs := imagesystem.BuilderObjects(name, system.ImagesNamespace, forNamespace, system.DefaultImage(), pubKey, privKey, builder.Status.UUID, registryDNS)
+	objs := imagesystem.BuilderObjects(name, system.ImagesNamespace, forNamespace, system.DefaultImage(), pubKey, privKey, builder.Status.UUID, registryDNS, *cfg.UseCustomCABundle)
```

Comment:
If `cfg.UseCustomCABundle` is `nil` when we reach this point we'll get a panic.

Example: https://go.dev/play/p/3mChCxI3yAa

Reply:
I think this is always initilized so that it is not nil. https://github.com/acorn-io/acorn/blob/2141a882513218dd845b1aa33ef9f9ad8928979f/pkg/config/config.go#L97

Reply:
Yep, that's right. Thanks for helping me to remember.